plugins/providers/virtualbox/driver/version_5_0.rb

```diff
@@ -17,19 +17,23 @@ def initialize(uuid)
         end
 
         def clear_forwarded_ports
-          args = []
-          read_forwarded_ports(@uuid).each do |nic, name, _, _|
-            args.concat(["--natpf#{nic}", "delete", name])
-          end
+          retryable(on: Vagrant::Errors::VBoxManageError, tries: 3, sleep: 1) do
```

Comment:
Looks like `read_forwarded_ports()` is already calling `execute()` with `retryable: true`:
```
execute("showvminfo", uuid, "--machinereadable", retryable: true)
```

Could it be that failures happen in the other `execute()` call below?